pygmt/tests/test_plot.py

```diff
@@ -152,7 +153,7 @@ def test_plot_sizes(data, region):
     fig.plot(
```

Comment:
Can you also rename other instances of 'sizes' to 'size' in the `""" """` docstring? E.g. "Plot the data using z as sizes." becomes "Plot the data using z as size.". Also debating whether to keep the test names as `test_plot_*_sizes` or rename to `test_plot_*_size` (which would mean renaming a lot of *.png.dvc files).

Reply:
> E.g. "Plot the data using z as sizes." becomes "Plot the data using z as size."

In this sentence, "sizes" means "sizes" of symbols, not the `size` parameter. I think it is grammatically correct, right? 

BTW, we also use "Plot the data using z as colors" even though the parameter name is `color`.

Reply:
Hmm, I think either `sizes` or `size` is grammatically correct. Let's just keep it on `sizes` to reduce the `git diff` then.